protocol/flows/ping/ping.go

```diff
@@ -11,9 +11,13 @@ import (
 	"github.com/kaspanet/kaspad/wire"
 )
 
+// Context is the interface for the context needed for the Ping and Pong flows.
```

Comment:
Maybe no need for Context if nothing in there?
Still might be good as a placeholder, not sure.

Anyway, if decide to keep it, please rename as in all other places [FlowName]Context